dependabot/update_dependencies_in_pipeline.py

```diff
@@ -188,34 +188,36 @@ def wait_for_current_level_build(level):
         chat_message += 'Following modules dependency PRs have failed checks...' + "\n"
         for module in pr_checks_failed_modules:
             print(module['name'])
-            chat_message += module['name'] + "\n"
+            chat_message += "<" + module[MODULE_CREATED_PR] + "|" + module['name'] + ">" + "\n"
 
     pr_merged_failed_modules = list(
         filter(lambda s: s[MODULE_CONCLUSION] == MODULE_CONCLUSION_PR_MERGE_FAILURE, current_level_modules))
     if len(pr_merged_failed_modules) != 0:
         module_release_failure = True
         chat_message += 'Following modules dependency PRs could not be merged...' + "\n"
         for module in pr_merged_failed_modules:
-            chat_message += module['name'] + "\n"
+            print(module['name'])
+            chat_message += "<" + module[MODULE_CREATED_PR] + "|" + module['name'] + ">" + "\n"
 
     build_checks_failed_modules = list(
         filter(lambda s: s[MODULE_CONCLUSION] == MODULE_CONCLUSION_BUILD_FAILURE, current_level_modules))
     if len(build_checks_failed_modules) != 0:
         module_release_failure = True
         chat_message += 'Following modules timestamped build checks failed...' + "\n"
         for module in build_checks_failed_modules:
-            chat_message += module['name'] + "\n"
+            print(module['name'])
+            chat_message += "<" + module[MODULE_CREATED_PR] + "|" + module['name'] + ">" + "\n"
```

Comment:
Let's give the link to the build actions page, for example, https://github.com/ballerina-platform/module-ballerina-io/actions/workflows/build-timestamped-master.yml